packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -234,8 +257,13 @@ export class Wallet {
     return result.messagesByObjective[objectiveId];
   }
 
+  public get messageService(): MessageServiceInterface {
```

Comment:
We only use this for tests but I think it's ok to expose anyways.

Reply:
Should `this._messageService` be `readonly`? With `readonly`, we might not need to define the getter.